app/views/broken_window/metrics/show.html.erb

```diff
@@ -13,10 +13,10 @@
       <div class="minimum"><%= @metric.threshold_type %> <%= BrokenWindow::Metric.format_value @metric.threshold, @metric.value_type %></div>
     </div>
     <%= tag :div, class: "chart", data: {
-      sparkline: @metric.measurements.order(id: :asc).map{ |measurement| {
+      sparkline: @metric.unique_daily_measurements.map{ |measurement| {
```

Comment:
@stepantubanov not sure about method name

Reply:
may be just `daily_measurements`?